https://github.com/libsdl-org/sdl/commit/e2ade2bfc46d

ld: error: undefined symbol: wl_proxy_marshal_flags
>>> referenced by SDL_waylanddatamanager.c
>>>               build/.libs/SDL_waylanddatamanager.o:(Wayland_data_source_destroy)
>>> referenced by SDL_waylanddatamanager.c
>>>               build/.libs/SDL_waylanddatamanager.o:(Wayland_data_offer_receive)
>>> referenced by SDL_waylanddatamanager.c
>>>               build/.libs/SDL_waylanddatamanager.o:(Wayland_data_offer_destroy)
>>> referenced 109 more times

--- src/video/wayland/SDL_waylanddyn.h.orig	2020-03-11 01:36:18 UTC
+++ src/video/wayland/SDL_waylanddyn.h
@@ -75,6 +75,8 @@ void SDL_WAYLAND_UnloadSymbols(void);
 #define wl_proxy_create (*WAYLAND_wl_proxy_create)
 #define wl_proxy_destroy (*WAYLAND_wl_proxy_destroy)
 #define wl_proxy_marshal (*WAYLAND_wl_proxy_marshal)
+#define wl_proxy_marshal_flags (*WAYLAND_wl_proxy_marshal_flags)
+#define wl_proxy_marshal_array_flags (*WAYLAND_wl_proxy_marshal_array_flags)
 #define wl_proxy_set_user_data (*WAYLAND_wl_proxy_set_user_data)
 #define wl_proxy_get_user_data (*WAYLAND_wl_proxy_get_user_data)
 #define wl_proxy_get_version (*WAYLAND_wl_proxy_get_version)
--- src/video/wayland/SDL_waylandsym.h.orig	2020-03-11 01:36:18 UTC
+++ src/video/wayland/SDL_waylandsym.h
@@ -71,6 +71,10 @@ SDL_WAYLAND_SYM(struct wl_proxy *, wl_proxy_marshal_co
 SDL_WAYLAND_MODULE(WAYLAND_CLIENT_1_10)
 SDL_WAYLAND_SYM(struct wl_proxy *, wl_proxy_marshal_constructor_versioned, (struct wl_proxy *proxy, uint32_t opcode, const struct wl_interface *interface, uint32_t version, ...))
 
+SDL_WAYLAND_MODULE(WAYLAND_CLIENT_1_20)
+SDL_WAYLAND_SYM(struct wl_proxy*, wl_proxy_marshal_flags, (struct wl_proxy *proxy, uint32_t opcode, const struct wl_interface *interfac, uint32_t version, uint32_t flags, ...))
+SDL_WAYLAND_SYM(struct wl_proxy*, wl_proxy_marshal_array_flags, (struct wl_proxy *proxy, uint32_t opcode, const struct wl_interface *interface, uint32_t version,  uint32_t flags, union wl_argument *args))
+
 SDL_WAYLAND_INTERFACE(wl_seat_interface)
 SDL_WAYLAND_INTERFACE(wl_surface_interface)
 SDL_WAYLAND_INTERFACE(wl_shm_pool_interface)
